quarkus-workshop-super-heroes/docs/src/docs/asciidoc/core-introduction/introduction-presentation.adoc

```diff
@@ -130,10 +140,30 @@ This workshop should be as self-explanatory as possible.
 So your job is to follow the instructions by yourself, do what you are supposed to do, and do not hesitate to ask for any clarification or assistance; that's why the team is here.
 
 You will download a zip file ({github-raw}/dist/quarkus-super-heroes-workshop.zip[`quarkus-super-heroes-workshop.zip`]) containing _some_ of the code and you will need to complete it.
-If you are stuck, you can always look at the final code in the {github-url}[GitHub repository], or download another zip file ({github-raw}/dist/quarkus-super-heroes-workshop-complete.zip[`quarkus-super-heroes-workshop-complete.zip`]) that contains _all_ the code of the workshop.
 
 Oh, and be ready to have some fun!
 
+[NOTE]
+====
+If you are stuck, you can always look at the final code in the {github-url}[GitHub repository], or download another zip file ({github-raw}/dist/quarkus-super-heroes-workshop-complete.zip[`quarkus-super-heroes-workshop-complete.zip`]) that contains _all_ the code of the workshop.
+If you want to build the completed workshop, after you've unzipped the {github-raw}/dist/quarkus-super-heroes-workshop-complete.zip[`quarkus-super-heroes-workshop-complete.zip`] file, you would need to run the following command from the root project:
+
+[source, shell]
+----
+$ ./mvnw clean install -Pcomplete
+----
+
+ifdef::use-extension[]
+The extension can be built with the following command:
+
+[source, shell]
+----
+$ ./mvnw clean install -f super-heroes/extension-version
+----
+endif::use-extension[]
+====
+
+
```

Comment:
I think we can remove all this content. That change happened in parallel with this one ... now that building the complete workshop is just `mvn install`, we don't need to document it, which means people don't end up auto-copy-pasting the wrong instructions.